HELM.md

```diff
@@ -111,8 +102,8 @@ By default services will be exposed with following hostnames (can be changed, se
 
 All control plane components can be cleaned up with helm with:
 
-```bash
-helm delete --purge openfaas
+```
```

Comment:
Why remove the `bash` indicator?

Reply:
there should bash indicators everywhere or not at all, just to keep code consistent :-)